docs/signage.md

```
@@ -0,0 +1,173 @@
+| POST   | `/api/v1/signage/content` | Create content |
+| POST   | `/api/v1/signage/content/{id}/publish` | Publish a draft |
+| GET    | `/api/v1/signage/playlists` | List playlists |
+| POST   | `/api/v1/signage/playlists` | Create a playlist |
+| GET    | `/api/v1/signage/playlists/{id}/items` | List items in a playlist |
+| POST   | `/api/v1/signage/playlists/{id}/items` | Add item to playlist |
+
+---
+
+## Data Model
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)